docs/configuration.rst

```diff
@@ -784,6 +784,14 @@ Storage Server Configuration
 
 .. _#390: https://tahoe-lafs.org/trac/tahoe-lafs/ticket/390
 
+``storage_dir = (string, optional)``
+
+    This specifies a storing directory.
+
```

Comment:
Maybe something like "This is where share files and other state pertaining to storage-servers will be kept".